pennylane/wires.py

```diff
@@ -129,6 +129,8 @@ def _unflatten(cls, data, _metadata):
         return cls(data, _override=True)
 
     def __init__(self, wires, _override=False):
+        if wires is None:
+            raise ValueError("Must specify a set of wires.")
```

Comment:
```suggestion
            raise ValueError("Must specify a set of wires. None is not a valid wire label.")
```
Maybe? I feel like that phrasing is a little passive aggressive, but I'm thinking we should communicate something more explicit about having received `None` and it not being an acceptable wire label.